docs/F-Refinements/portability.md

```diff
@@ -138,79 +141,78 @@ AIX compilers store char with a range of 0 to 255, while `gcc` and `cl` store ch
 int main(void)
 {
         int c;  // store character as an int
-
         c = getchar();
 
         if (c == EOF)
                 printf("End of data\n");
         else
-                printf("You entered %c\n", c); 
-
+                printf("You entered %c\n", c);
         return 0;
 }
 ```
 
-To identify this portability issue on an AIX platform, we can pass the source code through the lint code checker:
+To identify this portability issue on an **AIX** platform, we can pass the source code through the Lint code checker:
 
-```
+```text
 lint lint.c
 ```
 
-Example message from lint:
+Example message from Lint:
 
-```
-"lint.c", line 13: warning: comparison of unsigned with negative constant 
+```text
+"lint.c", line 13: warning: comparison of unsigned with negative constant
 ```
 
 To identify this portability issue on a `cl` platform, we change the compiler warning level options to `/W4` (warning level 4) and `/WX` (treat all warnings as errors).
 
-> Note<br/>
-> You can set these attributes in the Visual Studio project properties
+:::note Note
```

Comment:
I believe we can skip the title for the Admonition if we don't have a custom title.
Instead of having `:::note Note`, we can simply have `:::note`.